### ISSUE TITLE
Update Alderney Record Centre boundary

### ISSUE THREAD

**Issue by johnvanbreda** [BiologicalRecordsCentre/iRecord#1671]
@kitenetter the boundary for Alderney Records Centre is different to the one I've received from Matt Lewis at Alderney Wildlife Trust - see https://warehouse1.indicia.org.uk/index.php/location/edit/215429:
![image](https://github.com/BiologicalRecordsCentre/iRecord/assets/2392594/4bab0879-bfc7-4d91-8c15-e449aa4fbebb)

It also has some odd kinks in the line on the top edge if you zoom in. The one supplied by Matt aligns with the grey boundary that you can see on the OpenStreetMap background. 

Is it OK for me to update this?

**Comment by johnvanbreda** [comment]
Also the LRC boundary for Jersey looks very rough and could be updated at the same time (202728).

**Comment by kitenetter** [comment]
I can't remember where the Channel Islands LRC boundaries came from, I don't think they are part of the ALERC GIS layer. Which means that I don't think there is any difficulty in updating them, but ideally we should keep all three properly consistent. So if Alderney changes then Guernsey needs to change as well, and perhaps Jersey, to make all three consistent with the grey lines on the OSM background?

![image](https://github.com/BiologicalRecordsCentre/iRecord/assets/16039111/01e5045c-54fc-4840-8dc9-731952903f9f)

(The outermost OSM grey lines are the same boundaries as used by the Channel Isles vice-county boundary, so aligning with this will also help make things more consistent.)

Copying in @burkmarr in case he is aware of any other issues with the CI boundaries.

**Comment by burkmarr** [comment]
Those boundaries came to us from ALERC and were included in the complete update we did for this issue: https://github.com/BiologicalRecordsCentre/iRecord/issues/1633. If you look at that issue you will see that ALREC subsequently identified some problems with that update and sent a new shapefile to @robin-hutchinson. What isn't in the issue was that I found multiple problems with the new shapefile they sent which I detailed in an email to Robin.
